samples/python-event-requests/cloudevent_to_request.py

```diff
@@ -0,0 +1,39 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import sys
+import io
+from cloudevents.sdk.http_events import Event
+
+import requests
+
+if __name__ == "__main__":
+
+    if len(sys.argv) < 2:
+        sys.exit("Usage: python with_requests.py "
+                 "<CloudEvents controller URL>")
+
+    url = sys.argv[1]
+
+    # Configuration
+    headers = {
+        "ce-id": "my-id",
+        "ce-source": "<event-source>",
+        "ce-type": "cloudevent.event.type",
+        "ce-specversion": "0.2"
+    }
+    data = {"payload-content": "Hello World!"}
+
+    event = Event(headers=headers, data=data)
+    print(f"Sent {event}")
+    requests.post(url, headers=event.headers, json=event.data)
```

Comment:
I'm not seeing where this file is used. Is it just meant to be a sample? If so, we don't need to add `requests` as a test dependency, because this file is never invoked.

Reply:
Yeah its just sample code